src/test/java/healthy/lifestyle/backend/users/service/UserServiceTest.java

```diff
@@ -70,4 +76,33 @@ void createUserTest_shouldReturnUserDto() {
         assertEquals(1L, responseDto.getId());
         assertEquals(age, signupRequestDto.getAge());
     }
+
+    @Test
+    void updateUserTest_shouldReturnUserDto() {
+        UpdateUserRequestDto updateUserRequestDto = dataUtil.createUpdateUserRequestDto("one", 1L, 25);
+        Long userId = 1L;
+        Country country = Country.builder().id(1L).name("Country").build();
+        User existingUser = User.builder()
+                .username("oldUsername")
+                .email("oldEmail")
+                .fullName("oldFullName")
+                .age(35)
+                .country(country)
+                .build();
+
+        // When
+        when(userRepository.getReferenceById(userId)).thenReturn(existingUser);
+        when(countryRepository.getReferenceById(updateUserRequestDto.getUpdatedCountryId()))
+                .thenReturn(country);
+        when(userRepository.save(existingUser)).thenReturn(existingUser);
```

Comment:
when(...) relates to the initial setup of a test, so it's something given before the test. In When section there is a method under a test, so "When" in a given-when-then relates to the main test event.